rdt/transformers/datetime.py

```diff
@@ -30,6 +30,9 @@ class UnixTimestampEncoder(BaseTransformer):
         datetime_format (str):
             The strftime to use for parsing time. For more information, see
             https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior.
+        enforce_min_max_values (bool):
```

Comment:
Maybe we should add this as the last parameter. The main reason is that if someone runs scripts without using the key-word argument names, it might crash after this release since the 4th argument used to be a string and now it is a boolean with a different purpose. This isn't a huge deal though